x/rewards/keeper/flat_fee.go

```diff
@@ -37,3 +40,36 @@ func (k Keeper) GetFlatFee(ctx sdk.Context, contractAddr sdk.AccAddress) (sdk.Co
 
 	return fee, true
 }
+
+// ImportFlatFees initializes state from the genesis flat fees data.
+func (k Keeper) ImportFlatFees(ctx sdk.Context, flatFees []types.FlatFee) {
```

Comment:
Would be better to implement this in `state_flatfee.go` to maintain consistency.